common/wscript

```diff
@@ -142,6 +142,7 @@ def build(bld):
     clientlib.defines = 'HAVE_CONFIG_H'
     clientlib.use = uselib
     if bld.env['IS_WINDOWS']:
+         clientlib.env['cxxshlib_PATTERN'] = 'lib%s.dll'
```

Comment:
What happens if you put `bld.env['cxxshlib_PATTERN'] = 'lib%s.dll'` in the `if bld.env['IS_WINDOWS']` block just above and drop this and similar assignments in the `IS_WINDOWS` block below?